gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -59,17 +59,65 @@ async fn gatewayd_shows_info_about_all_connected_federations() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
-async fn gatewayd_shows_balance_for_any_connected_federation() -> anyhow::Result<()> {
-    // todo: implement test case
+#[ignore] // balance query needs mint module but that is not configured in fixtures
+async fn gatewayd_shows_balance_for_any_connected_federation() {
+    // TODO: Merge with `gatewayd_shows_info_about_all_connected_federations`
+    // after Issue #2508 is resolved
+    let (_, rpc, fed1, fed2, _) = fixtures::fixtures(None).await;
 
-    Ok(())
+    let id1 = fed1.connection_code().id;
+    let id2 = fed2.connection_code().id;
+
+    connect_federations(&rpc, &[fed1, fed2]).await.unwrap();
+
+    let bal1 = rpc
+        .get_balance(BalancePayload { federation_id: id1 })
+        .await
+        .unwrap();
+
+    assert_eq!(bal1.msats, 0);
+
+    let bal2 = rpc
+        .get_balance(BalancePayload { federation_id: id2 })
+        .await
+        .unwrap();
+
+    assert_eq!(bal2.msats, 0);
 }
 
 #[tokio::test(flavor = "multi_thread")]
-async fn gatewayd_allows_deposit_to_any_connected_federation() -> anyhow::Result<()> {
-    // todo: implement test case
+#[ignore] // deposit needs wallet module,
+          // balance query needs mint module,
+          // both are not configured in fixtures
+async fn gatewayd_allows_deposit_to_any_connected_federation() {
+    let (_, rpc, fed, _, bitcoin) = fixtures::fixtures(None).await;
 
-    Ok(())
+    let id = fed.connection_code().id;
+
+    connect_federations(&rpc, &[fed]).await.unwrap();
+
+    let addr = rpc
+        .get_deposit_address(DepositAddressPayload { federation_id: id })
```

Comment:
same here, except now we need the wallet module

Reply:
We will need to use the wallet module for testing deposits, but again better to wait for the NG client.

Reply:
ack.